api/queries_repo.go

```diff
@@ -59,6 +60,46 @@ func (r Repository) ViewerCanPush() bool {
 	}
 }
 
+func RepoExistsOnGitHub(client *Client, repo ghrepo.Interface) (bool, error) {
+	query := `
+	query($owner: String!, $name: String!) {
```

Comment:
This performs mostly the same lookup as `GitHubRepo`. Could this method call `GitHubRepo()` internally and simply do the `NOT_FOUND` error handling on top of that?

Reply:
done